commands/bootstrap_cli.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python
+
+from collections import defaultdict
+from pathlib import Path
+
+import click
+from cloudfoundry_client.client import CloudFoundryClient
+from schema import Optional, Schema
+import yaml
+
+from .utils import camel_case, get_ssm_secret_names, mkdir, mkfile, set_ssm_param, SSM_PATH, setup_templates
+
+
+config_schema = Schema({
+    "app": str,
+    "environments": {
+        str: {
+            Optional("certificate_arns"): [str]
+        }
+    },
+    "services": [
+        {
+            "name": str,
+            "type": lambda s: s in ("public", "backend",),
+            "repo": str,
+            "image_location": str,
+            Optional("notes"): str,
+            Optional("secrets_from"): str,
+            "environments": {
+                str: {
+                    "paas": str,
+                    Optional("url"): str,
+                    Optional("ipfilter"): bool,
+                }
+            },
+            Optional("backing-services"): [
+                {
+                    "name": str,
+                    "type": lambda s: s in ("s3", "external-s3", "postgres", "redis", "opensearch",),
+                    Optional("paas-description"): str,
+                    Optional("paas-instance"): str,
+                    Optional("notes"): str,
+                    Optional("bucket_name"): str,           # for external-s3 type
+                    Optional("readonly"): bool,             # for external-s3 type
+                    Optional("shared"): bool,
+                }
+            ],
+            Optional("overlapping_secrets"): [ str ],
+            "secrets": {
+                Optional(str): str,
+            },
+            "env_vars": {
+                Optional(str): str,
+            },
+        }
+    ],
+})
+
+
+def get_paas_env_vars(client, paas):
```

Comment:
I'm not sure how much this is just a reorganise or whether I should be asking awkward questions like...

Could we have unit tests for these methods?